.github/workflows/create-location-date-sequence-counts.yaml

```diff
@@ -0,0 +1,57 @@
+name: Create a file of sequence counts by location and collection date
+
+on:
+  schedule:
+    # Not precise, but GithHub actions don't support time zones
+    - cron: "0 02 * 11-12,1-3 3" # 2 AM UTC every Wed (Nov-Dec, Jan-Mar)
```

Comment:
Timezones 🙀 

Rounds close at 8 PM US Eastern, and GitHub actions are UTC, so tried to come as close as possible when accounting to daylight savings time (not exact, obviously)